students/jemielity_kamil/lesson_01_basics/tens_digit.py

```diff
@@ -0,0 +1,5 @@
+
+integer = int(input("Write an integer: "))
+
+print(str(integer)[-2])
```

Comment:
I believe this solution wasn't accepted by Snakify. Please add a comment if you can't find the correct solution.

Try to use 'integer // 2' in your code